pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -124,7 +124,7 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 
 	desiredService, err := resources.CreateKnativeService(kfsvc)
 	if err != nil {
-		log.Error(err, "Failed to create desired ksvc", "name", desiredService.Name)
+		log.Error(err, "Failed to create desired Knative Serving Service", "name", kfsvc.Name)
```

Comment:
If we're going to change this, we at least need to be consistent. KFService?

Reply:
I just re-read this XD. Nvm.

Reply:
I agree, as long as it is consistent whether it is full name or short name, ksvc is a short name for Knative Service.